keyform/views.py

```diff
@@ -1,10 +1,17 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+from django.views.generic import CreateView
 from django.views.generic.list import ListView
+from keyform.forms import CreateForm
 from keyform.models import Request
 
-
 class HomeView(ListView):
     model = Request
     template_name = "keyform/home.html"
+
+class NewForm(CreateView):
+    template_name = "keyform/add_form.html"
+    model = Request
+    success_url = "/"
```

Comment:
You should probably give the home url a name in keyform.urls.py:
`url(r'^$', views.HomeView.as_view(), name='home'),`
and then replace this with
`success_url = reverse_lazy('home')`
to avoid hard-coding the url.